src/storage/redis_db.cc

```diff
@@ -309,10 +314,12 @@ rocksdb::Status Database::Keys(const std::string &prefix, std::vector<std::strin
   return rocksdb::Status::OK();
 }
 
-rocksdb::Status Database::Scan(const std::string &cursor, uint64_t limit, const std::string &prefix,
-                               std::vector<std::string> *keys, std::string *end_cursor, RedisType type) {
+rocksdb::Status Database::Scan(const std::string &cursor, uint64_t scan_success_limit, uint64_t scan_false_limit,
```

Comment:
`scan_matched_limt` and `scan_mismatched_limit` should be a better name in this scenario.

Reply:
The parameter of the SCAN command is too long now, it's time to group the scan parameters into a struct.